samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -746,6 +761,117 @@ object SamzaContainer extends Logging {
       containerStorageManager = containerStorageManager)
   }
 
+
+  private[container] def getStandbyTasks(jobModel: JobModel, containerModel: ContainerModel,
```

Comment:
Formatting: For this particular case I'd prefer param per line since there are so many of them. Same at the call site.